package.json

```diff
@@ -24,6 +24,7 @@
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
     "clsx": "^1.2.1",
+    "fast-xml-parser": "^4.2.2",
```

Comment:
Is there a reason we're using xml over json?

Reply:
the generated jest-junit report is xml - just included the parser for the mini script